src/msg.h

```diff
@@ -87,6 +87,7 @@
 #define MSG_CFG_INTERNAL_FAIL   MSG_PREFIX "0071E" " Internal failure during validation, please contact support\n"
 #define MSG_CFG_LOAD_FAIL       MSG_PREFIX "0072E" " Launcher Could not load configurations\n"
 #define MSG_CFG_SCHEMA_FAIL     MSG_PREFIX "0073E" " Launcher Could not load schemas, status=%d\n"
+#define MSG_NO_LOG_CONTEXT      MSG_PREFIX "0074E" " Log context was not created\n"
```

Comment:
Just a question, are these messages supposed to be in docs-site somewhere?

Reply:
Yes, the messages should get there. Can we create issue in this repository to do it at some point?

Reply:
I guess it should be at https://docs.zowe.org/stable/troubleshoot/launcher/launcher-error-codes/#zwel0005i, but it ends with ZWEL0047E for some reason.

Reply:
Seems outdated, an issue in docs-site sounds good, but since this comes directly from the code it'd be better to propose a draft of the changes.